### PR TITLE
chore: v0.9.0

### PR DESCRIPTION
**Pull request by Sanix-Darker**

### CODE REVIEW

**Review by fspot — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+   -  bump chardet from 4.0.0 to 5.0.0
+   -  bump types-pyyaml from 6.0.8 to 6.0.9
+   -  bump types-paramiko from 2.11.0 to 2.11.1
+   -  bump geopandas from 0.10.2 to 0.11.0
+   -  bump mkdocs-material from 8.3.3 to 8.3.6
+   -  bump types-paramiko from 2.10.0 to 2.11.0
+   -  bump certifi from 2022.5.18.1 to 2022.6.15
+   -  bump actions/setup-python from 3 to 4
+   -  bump jq from 1.2.3 to 1.3.0
+   -  bump docker from 5.0.3 to 6.0.0
```
Comment (lukapeschke) [inline CHANGELOG.md]: I don't think this is relevant for end users. The only change that could impact them is `chardet` bumped from `^4.0.0` to `>=4,<6`, meaning they could end up with chardet 5 if they did not pin it. However, if they're using it , it should be pinned by them anyway so I don't think it is relevant to mention it either

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+- fix(ci): Pin python version to 3.10.6
```
Comment (lukapeschke) [inline CHANGELOG.md]: Not relevant either. Also, I'd like to bump mypy to `0.981` before making a release

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## [0.8.5] -   2022-09-08
```
Comment (lukapeschke) [inline CHANGELOG.md]: Dropping support for python < 3.10 is breaking, so this should be 0.9.0

**Review by lukapeschke — CHANGES_REQUESTED** [review]
Thanks for taking care of this. But is a release really needed ? No features have been shipped

**Review by lukapeschke — APPROVED** [review]
:100:

**Pull request merged**